### PR TITLE
Vietnamese translation

### PR DESCRIPTION
**Pull request by thuvh**
Finish Vietnamese translation base on pr #34. This is a word by word translation. This book should be improved in the future, to make a fluently translation.

### CODE REVIEW

**Comment by davidrouten** [comment]
Hi @thuvh! Sorry for the belated response and thank you for taking the time to translate these pages.

We love to have one reviewer per translation if we can - do you happen to have a colleague that can proof-read your PR?

**Comment by thuvh** [comment]
@joneslee85 could you please help to review?

File: content/vi/toc.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Dịch vụ hỗ trợ như là tài nguyên bổ xung
+
+## [V. Xây dựng, phát hành, vận hành](./build-release-run)
+### Tách biệt hoàn toàn giữa bước xây dựng và vận hành
+
+## [VI. Tiến trình](./processes)
+### Vận hành ứng dụng như là một hoặc nhiều tiến trình phi trạng thái 
+
+## [VII. Mở cổng mạng](./port-binding)
+### Cung cấp các dịch vụ thông qua công mạng
```
Comment (joneslee85) [inline content/vi/toc.md]: typo co^ng, it should be co^?ng

File: content/vi/toc.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## [IV. Dịch vụ hỗ trợ](./backing-services)
+### Dịch vụ hỗ trợ như là tài nguyên bổ xung
+
+## [V. Xây dựng, phát hành, vận hành](./build-release-run)
+### Tách biệt hoàn toàn giữa bước xây dựng và vận hành
+
+## [VI. Tiến trình](./processes)
+### Vận hành ứng dụng như là một hoặc nhiều tiến trình phi trạng thái 
+
+## [VII. Mở cổng mạng](./port-binding)
```
Comment (joneslee85) [inline content/vi/toc.md]: I think this translated term is a bit ambiguous. Your translation literally means "Opening Ports".  What do you think about "Gán cổng"?

File: content/vi/admin-processes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+* Áp dụng các thay đổi cho cơ sở dữ liệu (như là `manage.py migrate` với Django, `rake db:migrate` với Rails).
+* Giao diện điều khiển trực tiếp ứng dụng (được biết đến như là vỏ [REPL](http://en.wikipedia.org/wiki/Read-eval-print_loop)) để vận hành các đoạn mã nguồn tuỳ ý hoặc kiểm tra các mô hình của ứng dụng đối với cơ sở dữ liệu hiện tại. Hầu hết các ngôn ngữ cung cấp REPL bằng cách thực thi một bộ biên dịch không có tham số như là (e.g. `python` or `perl`) hoặc trong vài trường hợp có các câu lệnh đặc biệt (như là `irb` với Ruby, `rails console` với Rails).
+* Thực thi một lần đoạn kịch bản được quản lý trên kho mã nguồn của ứng dụng (như là `php scripts/fix_bad_records.php`).
+
+Tiến trình quản trị thực thi một lần nên đươcj vận hành trong một môi trường giống như [tiến trình vận hành lâu dài](./processes) của ứng dụng. Với một bản [phát hành](./build-release-run), các tiến trình quản trị sử dụng cùng [mã gốc](./codebase) và [cấu hình](./config) như bất kỳ các tiến trình vận hành trong bản phát hành đó. Đoạn mã quản trị cần được triển khai cùng với mã của ứng dụng để đảm bảo không có các vấn đề về mặt động bộ hoá.
```
Comment (joneslee85) [inline content/vi/admin-processes.md]: s/đươcj/được

**Comment by joneslee85** [comment]
LGTM 👍 

Few nitpicks and typos to address. Then it is good to merge

File: content/vi/toc.md (hunk 2 above)
Comment (thuvh) [reply content/vi/toc.md]: I translated word by word. "Gán cổng" is better.

**Comment by joneslee85** [comment]
@thuvh literal translation of technical terms could lead to confusion because many would have been used to English word. I reckon you should have both English term and translated term next to each other, for example Gán cổng (Port binding)

**Pull request merged**